### PR TITLE
Review html page names in SDK docs - closes #532

### LINKED ISSUE #532: Review html page names
## Which content is outdated?
Review, if the file names of the docs pages can be improved.
example: API specification has api.html, configuration has config.html, Create custom transaction types has customize.html, reference to refrerences.

- [x] Review file names in latest Core docs
- [x] Review file names in latest SDK docs
- [x] Review file names in 2.1.6 Core docs

### PR DESCRIPTION
**Pull request by chris529**
Closes issue #532

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -1,27 +1,27 @@
 * xref:setup.adoc[Prerequisites]
 * xref:guides/index.adoc[Guides]
 ** xref:guides/configuration.adoc[Configure a blockchain application]
-** xref:guides/customize.adoc[Create a custom transaction]
-** xref:guides/test.adoc[Test a custom transaction]
-** xref:guides/interact.adoc[Interact with the API]
+** xref:guides/customize-transactions.adoc[Create a custom transaction]
```
Comment (Tschakki) [inline modules/ROOT/nav.adoc]:
```suggestion
** xref:guides/custom-transactions.adoc[Create a custom transaction]
```

File: modules/ROOT/nav.adoc [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
-*** xref:tutorials/transport0.adoc[Part 0: Installation & setup]
-*** xref:tutorials/transport1.adoc[Part 1: Track a packet on the blockchain]
-*** xref:tutorials/transport2.adoc[Part 2: A simple supply chain management system]
-*** xref:tutorials/transport3.adoc[Part 3: How to publish the application]
+** xref:tutorials/supply-chain/index.adoc[Supply chain]
+*** xref:tutorials/supply-chain/part1.adoc[Part 1: Installation & setup]
+*** xref:tutorials/supply-chain/part2.adoc[Part 2: Track a packet on the blockchain]
+*** xref:tutorials/supply-chain/part3.adoc[Part 3: A simple supply chain management system]
+*** xref:tutorials/supply-chain/part4.adoc[Part 4: How to publish the application]
 * xref:reference/index.adoc[References]
```
Comment (Tschakki) [inline modules/ROOT/nav.adoc]:
```suggestion
* xref:references/index.adoc[References]
```

I think we wanted to rename this to `references`.

File: modules/ROOT/nav.adoc [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
-** xref:reference/api.adoc[API specification]
+** xref:reference/api-specification.adoc[API specification]
 ** xref:reference/config.adoc[Configuration]
```
Comment (Tschakki) [inline modules/ROOT/nav.adoc]:
```suggestion
** xref:references/configuration.adoc[Configuration]
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
Looking good already, just a few suggestions :)

File: modules/ROOT/nav.adoc (hunk 2 above)
Comment (chris529) [reply modules/ROOT/nav.adoc]: Yes, well spotted Mona 👍

File: modules/ROOT/nav.adoc [hunk 4]
```diff
@@ -1,27 +1,27 @@
...
+** xref:reference/api-specification.adoc[API specification]
+** xref:references/configuration.adoc[Configuration]
 ** xref:reference/changelog.adoc[Developer changelog]
 ** xref:reference/lisk-commander/index.adoc[Lisk Commander]
 *** xref:reference/lisk-commander/commands.adoc[Commands]
-*** xref:reference/lisk-commander/lisk-core.adoc[Lisk Core commands]
+*** xref:reference/lisk-commander/lisk-core-commands.adoc[Lisk Core commands]
 *** xref:reference/lisk-commander/sensitive-inputs.adoc[Sensitive inputs]
 ** xref:reference/lisk-elements/index.adoc[Lisk Elements]
 *** xref:reference/lisk-elements/api-client.adoc[API client]
```
Comment (Tschakki) [inline modules/ROOT/nav.adoc]: Please also fix the links for the other pages and rename the `reference` folder to `references`

File: modules/ROOT/pages/guides/customize-transactions.adoc [hunk 5]
```diff
@@ -1,7 +1,7 @@
 = Create a custom transaction
 Mona Bärenfänger <mona@lightcurve.io>
 :description: The Lisk customization page explains how to create custom transactions and how to register them with the blockchain application.
-:page-aliases: customize.adoc
+:page-aliases: customize-transactions.adoc
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/customize-transactions.adoc]: This page alias should not be changed

File: modules/ROOT/pages/guides/index.adoc [hunk 6]
```diff
@@ -7,7 +7,7 @@ Mona Bärenfänger <mona@lightcurve.io>
 :url_react: https://reactjs.org/
 
 :url_config: guides/configuration.adoc
-:url_custom: guides/customize.adoc
+:url_custom: guides/customize-transactions.adoc
```
Comment (Tschakki) [inline modules/ROOT/pages/guides/index.adoc]: After renaming the filename of the page to `custom-transactions.adoc`, you also need to update all links referring to it.

**Review by Tschakki — CHANGES_REQUESTED** [review]
Please go back to this and rename the files/ folders everywhere consistently, and also update the links everywhere afterwards accordingly.

File: modules/ROOT/nav.adoc (hunk 4 above)
Comment (chris529) [reply modules/ROOT/nav.adoc]: OK, this should now be OK 👍

File: modules/ROOT/pages/guides/customize-transactions.adoc (hunk 5 above)
Comment (chris529) [reply modules/ROOT/pages/guides/customize-transactions.adoc]: OK, I am not 100% sure on this one Mona, I presume it should be customize.adoc then..?

File: modules/ROOT/pages/guides/index.adoc (hunk 6 above)
Comment (chris529) [reply modules/ROOT/pages/guides/index.adoc]: OK, done 👍

**Review by Tschakki — DISMISSED** [review]
Looks good!

**Pull request closed without merging**